angle_min : -1.91986
angle_max : 1.91986
angle_increment : 0.005759587
range_min : 0
range_max : 25
laser_frequency : 15

